src/boundingboxes.jl

```diff
@@ -40,6 +40,17 @@ end
 
 boundingbox(g::Grid) = Box(extrema(g)...)
 
+boundingbox(m::SimpleMesh) = boundingbox(vertices(m))
+
+function boundingbox(p::PolyArea)
+  box = boundingbox(vertices(p.outer))
+  for inner in p.inners
+      box = boundingbox(vertices(inner), coordinates(box.min), coordinates(box.max))
+  end
+  box
+end
```

Comment:
boundingbox(p::Polytope) = boundingbox(vertices(p))

Reply:
Yes, I am aware of that, however this forces unnecessary allocations, as vertices() of a polyarea has to collect all inner chains into an intermediate long vector of points. This saves these allocations.

Another optimization could be completely ignoring the inner vertices, as they are always inside the outer ring. But this could cause issues on malformed polygons

Reply:
The optimization with just the outer ring seems reasonable. Can we use that instead? We assume by design that inner rings are in the interior of the region defined by the outer ring.